Pipfile

```diff
@@ -51,11 +51,13 @@ pytest-cov = "*"
 pytest-mock = "*"
 pytest-timeout = "*"
 # Tools
+eth-ape = ">=0.6.3,<0.7.0"
+ape-solidity = ">=0.6.0,<0.7.0"
 hypothesis = "*"
 pre-commit = "2.12.1"
 pyflakes = "*"
 coverage = "<=6.5.0"
-py-solc-x = "==0.10.1"
+py-solc-x = "==1.1.1"   # docs version matches this one
```

Comment:
this dependency can be removed since it will be managed by eth-ape.

Reply:
Should be gone now.